vast/integration/integration.py

```diff
@@ -626,6 +638,11 @@ def main():
         type=Path,
         help="The basedir for the test runs",
     )
+    parser.add_argument(
+        "--dry-run",
```

Comment:
Should this be named `--list-suites` instead?

Reply:
I think `--list-suites` doesn't convey that the `--test` `--disable` and `--tag` options would be considered. `--dry-run` is commonly used for showing what would be done without actually doing it, so I think it fits quite well.